package.json

```diff
@@ -34,8 +34,11 @@
     "@akashnetwork/dev-config": "*",
     "@commitlint/cli": "^19.5.0",
     "@commitlint/config-conventional": "^19.5.0",
+    "@release-it/conventional-changelog": "github:ygrishajev/conventional-changelog#feature/pre-release",
     "cross-env": "^7.0.3",
     "husky": "^9.1.6",
+    "mvm": "^0.2.0",
```

Comment:
Minecraft version mananger is 11 years old and not maintained, is this a package you've used in the past?

Reply:
Is it this repo? https://www.npmjs.com/package/mvm https://www.npmjs.com/package/mvm

Reply:
Wow, actually found it on some article, and didn't bother much. I'll recheck the source code. It's a pretty simple package so maybe it's fine.

Reply:
Ok, I messed up. This is the package https://github.com/b12k/mvm

I'll remove the one commented